packages/admin/admin/src/dataGrid/excelExport/DataGridExcelExportButton.tsx

```diff
@@ -0,0 +1,44 @@
+import { Button, CircularProgress } from "@mui/material";
+import * as React from "react";
+import { FormattedMessage } from "react-intl";
+
+import { FileIcon } from "../../fileIcons/FileIcon";
+import { ExportApi } from "./useDataGridExcelExport";
+
+interface Props {
+    exportApi: ExportApi;
+    onClick?: () => void;
+    loadingComponent?: React.ReactNode;
+    onError?: () => void;
+}
+
+export const DataGridExcelExportButton: React.FunctionComponent<Props> = ({ onClick, children, exportApi, loadingComponent, onError }) => {
```

Comment:
On second thought: I don't actually think this button is necessary at all. This functionality should be inside a "More Actions" menu.

So you can remove this button altogether and replace it with "More actions" in the example